js/scripts.js

```diff
@@ -1,11 +1 @@
-function hideShowInstallationText(which) {
-  if (!document.getElementById) {
-    return;
-  }
-  if (which.style.display=="block") {
-    which.style.display="none";
-  }
-  else {
-    which.style.display="block";
-  }
-}
+//Nothing here
```

Comment:
I will remove this file, once my other penging PR is merged.
Otherwhise we would have to handle merging conflicts